spec/requests/api/v0.1/trainees/delete_degree_spec.rb

```diff
@@ -29,6 +29,8 @@
         )
         expect(response.parsed_body["data"]).to be_present
         expect(trainee.reload.degrees.count).to be_zero
+        expect(response.parsed_body.dig(*%w[data first_names])).to eq(trainee.first_names)
+        expect(response.parsed_body.dig(*%w[data last_name])).to eq(trainee.last_name)
```

Comment:
Maybe a check for `trainee_id` would be good here as it's excluded from all but the Trainee serializer?